routes/examples/paginationExample.js

```diff
@@ -0,0 +1,74 @@
+const router = require("express").Router();
+const Product = require("../../models/ProductModel");
+const authenticateUser = require("../verifyToken");
+
+router.get("/get-products", async (req, res) => {
+  try {
+    const limit = req.query.limit || 200;
```

Comment:
The limit is working correctly, I tried creating more than 300.
But, there is no way to fetch next set of records. Please refer to this video. It helped me in understanding the implementation of pages in pagination. https://youtu.be/rK1Dlu0cwgg

Reply:
Appreciate it.